src/components/react_multi_select.js

```diff
@@ -171,7 +172,7 @@ export default class ReactMultiSelect extends PureComponent {
         listHeight={selectedListHeight}
         listRowHeight={listRowHeight}
         emptyText={messages[DESTINATION_NO_ITEMS]}
-        displayFn={displayFn => displaySelectedItem(displayFn, this.props)}
+        displayFn={selectedItemRenderer  ? item => selectedItemRenderer(item) : displayFn => displaySelectedItem(displayFn, this.props)}
```

Comment:
This is an odd implementation.
Lets work on it after this PR.
I think it should be the same solution.